README.md

```diff
@@ -179,6 +179,13 @@ Called when an http response is received from the source.
 The default behavior is `reply.send(res)`, which will be disabled if the
 option is specified.
 
+#### `onError(reply, { error })`
```

Comment:
Please remove the wrapping object.

Reply:
@mcollina do you want to remove the entire wrapping object or just in the readme?

Reply:
``onError(reply, error)` < this should be the signature of this method.